tools/lib.webpack.config.babel.js

```diff
@@ -0,0 +1,124 @@
+const path = require('path')
+const webpack = require('webpack')
+const babelConfig = require('../.babelrc.js')
+const getVersion = require('git-repo-version')
+const nodeExternals = require('webpack-node-externals')
+const GeneratePackageJsonPlugin = require('generate-package-json-webpack-plugin')
+
+const basePackageValues = {
+  'name': '@integreat-app/shared',
+  'version': '0.0.4',
+  'main': './index.js'
+}
+
+const versionsPackageFilename = path.resolve(__dirname, '../package.json')
+
+module.exports = (env = {}) => {
+  if (env.prod === undefined) {
+    throw Error('You need to specify a mode!')
+  }
+
+  const isDebug = env.prod === 'false'
+  console.log('isDebug: ', isDebug)
+
+  return {
+    mode: 'production',
+    resolve: {
+      modules: [
+        path.resolve('./node_modules')
+      ]
+    },
+    // The base directory for resolving the entry option
+    context: path.resolve(__dirname, '../src'),
+    // The entry point for the bundle
+    entry: ['./lib.js'],
+
+    externals: [nodeExternals()],
+    // Options affecting the output of the compilation
+    output: {
+      path: path.resolve(__dirname, '../lib-dist'),
+      publicPath: '/lib-dist/',
+      filename: 'index.js',
+      sourcePrefix: '  ',
+      library: '@integreat/shared',
+      libraryTarget: 'umd'
+    },
+    // Developer tool to enhance debugging, source maps
+    // http://webpack.github.io/docs/configuration.html#devtool
+    devtool: isDebug ? 'source-map' : false,
+    // What information should be printed to the console
+    stats: 'verbose',
+    // The list of plugins for Webpack compiler
+    plugins: [
+      new GeneratePackageJsonPlugin(basePackageValues, versionsPackageFilename),
+      new webpack.DefinePlugin({
+        'process.env.NODE_ENV': '"production"',
```

Comment:
Same as above

Reply:
There are no dev builds of the lib currently, but I'll adjust it such that it is possible